### ISSUE TITLE
Topfiles not removed after release

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7339]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#7339|
|Type|task|
|Created|2014-05-12 17:20:19Z|

The file [twisted/web/topfiles/4888.bugfix](twisted/web/topfiles/4888.bugfix) is still present although it was part of 14.0 released hours ago (I think because its included in [twisted/web/topfiles/NEWS](twisted/web/topfiles/NEWS)). Shouldn't that file be removed during the release merge to trunk? Are there more leftover files like this?

<details><summary>Searchable metadata</summary>

```
trac-id__7339 7339
type__task task
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1399915219000000 1399915219000000
changetime__1422644835037478 1422644835037478
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

14.0.0 happened already.  Nothing can be added to that milestone now.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

I suspect this is not a general bug in the release toolchain but instead user error during the 14.0.0 release process - which had many strange quirks not actually supported by our tools.

So ... which of those strange quirks leads to not all news fragments being deleted?  It may be difficult to reconstruct the exact steps of the release process at this point.  Perhaps hawkowl can do it, though.

I'm going to suggest that this isn't a 14.1 blocker though because we should be able to avoid this issue by not doing the crazy merge-revert-merge-revert things that happened during 14.0.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

This was because I made a new merged forward branch to get 14.0 back into trunk, leaving the 14.0 one intact.

I think I can just delete this single stray file (I can't find any others) when I do the 14.1 release, before I generate the newsfiles. Then when 14.1 goes back into trunk, it'll fix up this ticket.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

Or we can delete it before 14.1.  No reason to make this *part* of the release process.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl set status to `closed`|
|-|-|

This is all fixed from the 15.0 release.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun commented|
|-|-|

And the general issue is fixed too?
